packages/nzyme/PKGBUILD

```diff
@@ -0,0 +1,29 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=nzyme
+pkgver=1.1.1
+pkgrel=1
+pkgdesc='WiFi defense system.'
+arch=('any')
+groups=('blackarch' 'blackarch-wireless' 'blackarch-defensive')
+url='https://www.nzyme.org/'
+license=('SSPL')
```

Comment:
```diff
-license=('SSPL')
+license=('custom:SSPL')
```

This one is not in the licenses package neither in the custom cases of https://wiki.archlinux.org/title/PKGBUILD#license